Sources/Testing/ABI/ABI.swift

```diff
@@ -18,7 +18,7 @@ extension ABI {
   /// A protocol describing the types that represent different ABI versions.
   protocol Version: Sendable {
     /// The numeric representation of this ABI version.
```

Comment:
nit: Can we generalize this description now since it's not a trivial number anymore?

Reply:
What do you want to change it to?